src/controllers/API.js

```diff
@@ -162,6 +167,12 @@ class APIController {
         response.included, "beforeRender", registry, frameworkReq, frameworkRes
       );
 
+      // Dasherize ougoing response
+      // (DELETE responses don't have primary data)
+      if (response.primary) {
+        response.primary = formatters.dasherizeResourceOrCollection(response.primary, registry);
```

Comment:
Here, we're going to need to dasherize the `response.included` collection as well. That makes me think that we should do this in the `Document` type, which currently serves as the response formatter and has this nice [`resourceToJSON`](https://github.com/andrewbranch/json-api/blob/master/src/types/Document.js#L99) function. At some point, we'll want to split the formatting logic out of document, probably, but for now, I think it's good to put all the formatting logic there, so it's in one place.